TheBTeam.BLL/Services/Edits.cs

```diff
@@ -0,0 +1,297 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TheBTeam.BLL.Model;
+using TheBTeam.BLL.Validators;
+
+namespace TheBTeam.BLL.Services
+{
+    public class Edits
+    {
+        const int MinAddressLength = 3;
+        const int MinNameLength = 2;
+        private const int MinPhoneNumberLength = 12;
+        const int MinAge = 13;
+        const int MaxAge = 99;
+        public static void EditUser(User user)
+        {
+            var tempuser = new User(user.FirstName, user.LastName, user.Gender, user.Age, user.Email, user.Phone, user.Address, user.Company, user.Currency, user.Balance);
+            string options = "1)Edit surname\n2)Edit age\n3)Edit Company\n4)Edit gender\n5)Edit phone\n6)Edit balance\n7)Edit currency\n8)Finish";
+            bool exit = false;
+            while (exit == false)
+            {
+                Console.WriteLine(options, "\n");
+                string choice = Console.ReadLine();
+                if (choice == "1")
+                {
+                    tempuser.LastName = GetStringInput("Last Name", MinNameLength);
+                }
+                else if (choice == "2")
+                {
+                    tempuser.Age = GetIntInput("Age", MinAge, MaxAge);
+                }
+                else if (choice == "3")
+                {
+                    tempuser.Company = GetCompany();
+                }
+                else if (choice == "4")
+                {
+                    tempuser.Gender = GetGender();
+                }
+                else if (choice == "5")
+                {
+                    tempuser.Phone = GetPhoneNumber();
+                }
+                else if (choice == "6")
+                {
+                    tempuser.Balance = GetDecimalInput("Current balance");
+                }
+                else if (choice == "7")
+                {
+                    tempuser.Currency = GetCurrency();
+                }
+                else if (choice == "8")
+                {
+                    Console.WriteLine(" S to save, X to cancel\n");
+                    bool choiceSubmitted = false;
+                    while (choiceSubmitted == false)
```

Comment:
maby do while will be more appropriate here

Reply:
refactored those weird loops into do while's and while true's.